code/game/gamemodes/cult/ritual.dm

```diff
@@ -42,13 +42,13 @@
 	if(!ticker.mode)
 		icon_state = "tome"
 	else
-		icon_state = ticker.mode.cultdat.tome_icon
+		icon_state = ticker.cultdat.tome_icon

```

Comment:
This should have been a damn proc, not spamming `ticker.cultdat.var_name_here` all over the damn place.

Reply:
my bad.